### PR TITLE
Page elements resolver added

### PR DESCRIPTION
**Pull request by patgrudniewski**

### CODE REVIEW

File: src/page/renderer/field/inputGenerator.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export default (type, baseProps) => props => <input { ...props } { ...baseProps } type={ type } />;
```
Comment (bartij) [inline src/page/renderer/field/inputGenerator.js]: why it has a special generator in a new file?
can't it be just:
```javascript
case 'text':
   return props => <input { ...props } { ...baseProps } type={ type } />;
```
in field.js ?

File: src/page/renderer/field.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (bartij) [inline src/page/renderer/field.js]: again :P filename field doesn't say anything specific to me. Maybe it's a field of rice? 🍚 😄

File: src/page/renderer/wrapper.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (bartij) [inline src/page/renderer/wrapper.js]: again filename

File: src/page/renderer/field.js (hunk 2 above)
Comment (bartij) [reply src/page/renderer/field.js]: resolved on private conversation

File: src/page/renderer/field/inputGenerator.js (hunk 1 above)
Comment (bartij) [reply src/page/renderer/field/inputGenerator.js]: resolved on private conversation

File: src/page/renderer/wrapper.js (hunk 3 above)
Comment (bartij) [reply src/page/renderer/wrapper.js]: resolved on private conversation

**Review by bartij — APPROVED** [review]

File: src/page/resolver/wrapperResolver.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default (type) => {
+  switch(type) {
+    case 'add':
+      return 'form';
+    case 'edit':
+    case 'list':
+    case 'show':
+      return 'div';
```
Comment (auamarto) [inline src/page/resolver/wrapperResolver.js:10]: don't see test for this case.

**Review by auamarto — CHANGES_REQUESTED** [review]

File: src/page/resolver/wrapperResolver.js (hunk 4 above)
Comment (patgrudniewski) [reply src/page/resolver/wrapperResolver.js:10]: it's temporary due to other tests. should be throwing exception but then other tests are not passing due to dependency to this resolver

**Review by auamarto — APPROVED** [review]

**Pull request merged**